app/forms/journeys/additional_payments_for_teaching/eligible_itt_subject_form.rb

```diff
@@ -31,7 +31,7 @@ def self.qualification_to_substring(qualification_symbol)
       end
 
       def available_subjects
-        @available_subjects ||= subject_symbols.map(&:to_s)
+        @available_subjects ||= subject_symbols.map(&:to_s).sort
```

Comment:
i don't think you need the `sort` on the end as method already sorts?

Reply:
For some reason one of the tests was failing without it 🤷 I couldn't work out why